meetups/2020/jan/25/sponsor_pitch.md

```diff
@@ -0,0 +1,94 @@
+
+CODE N BUILD is one of a kind hackathon that goes forth with the aim of encouraging healthy competition in order to incubate futuristic ideas.
+
+CODE N BUILD focuses on providing students and upcoming entrepreneurs with a platform to develop, compute, network and present their ideas to solve real-world problems.
+
+Our aim to nurture young minds can be assisted with your help. In return for your support, expertise, and guidance, we have some exciting marketing avenues! Gain an immediate competitive advantage for your business to ensure that your brand is well-known within the youth and student community. If you are looking to build awareness of your brand in the student sector, it doesn’t get bigger and better than Code N Build.
+
+- 500-600 Developers/ innovators (All over India).
+- 29 States.
+- Problem statements by big MNC's.
+- 30 Hours.
+- More than 8 Community Partners.
+
+- What: First- ever Hackathon organized with MNC in University by students.
+- Who: 450+ talented performance, developers, engineers, data scientists, project managers etc from across various colleges/universities of India.
+- When: January 25-26 2020
+- Where: JECRC University, Jaipur
+- Why: Innovation, Collaboration, Recruiting, Growth
+
+The event will be organized on 25th Jan 2020. It is basically a 30 hours hackathon. The promotions will be done all over India. Apart from solving the problem, there will be more to look out for like fun activities, quizzes, etc throughout the event.
+
+## Why CodeNBuild?
+
+- Communication and Networking Opportunities.
+- Working under pressure in real-life scenarios.
+- Teamwork under pressure
+- Healthy competitive environment
+- Exposure and Interaction with Professionals
+- Opportunity to deal with real problems
+- Developing an idea
+- Showcasing their creativity
+- Fun activities to relax and reboot
+- Opportunity to self learn and express
+- Interacting with fellow participants to learn and bond
+- Potential for the ideas to be developed as a project
+- Free registrations
+- Huge prize money
+- Many more.
+
+## What is a hackathon?
+
+A hackathon is a beehive for the aspiring developers and coders. A hub of
+brainstorming, a gathering, a platform, an opportunity for programmers, graphic
+designers, interface designers, project managers, and others to work and solve
+real-life problems in a short period of time.
+
+## What do we bring to you?
+
+A Pan India event, by bringing together passionate participants.
+
+Last year we had an immensely successful local hackathon for the JECRCians at JU.
+
+To take it to the next level, DSC has come forth in association with JU to make this event an unforgettable one for all of the participants.
+
+## Problem statement
+
+Based on tracks like:
+
+- ML/AI
+- Blockchain
+- Computer Vision
+- Fintech
+- General
+- IoT
+- EdTech
+- Retail
+- AR/VR,
+- RPA
```

Comment:
It would be nice to include full forms for these things. Not everyone knows that this is Robotic Process Automation

Reply:
ohh yes ill edit